tests/integration/test_authd/force_options/test_authd_force_options.py

```diff
@@ -0,0 +1,201 @@
+from wazuh_testing.tools.configuration import load_wazuh_configurations, set_section_wazuh_conf, write_wazuh_conf
+from wazuh_testing.tools.file import read_yaml
+from wazuh_testing.wazuh_db import query_wdb
+from authd import create_authd_request, validate_authd_response, AUTHD_KEY_REQUEST_TIMEOUT
+
+
+# Data paths
+data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(data_path, 'configuration.yaml')
+tests_path = os.path.join(data_path, 'test_cases')
+
+# Configurations
+configurations = load_wazuh_configurations(configurations_path, __name__)
+configuration_ids = ['authd_force_options']
+
+# Tests
+tests = []
+test_case_ids = []
+for file in os.listdir(tests_path):
+    group_name = file.split(".")[0]
+    file_tests = read_yaml(os.path.join(tests_path, file))
+    tests = tests + file_tests
+    test_case_ids = test_case_ids + [f"{group_name} {test_case['name']}" for test_case in file_tests]
+
+# Variables
+log_monitor_paths = []
+
+receiver_sockets_params = [(("localhost", 1515), 'AF_INET', 'SSL_TLSv1_2')]
+monitored_sockets_params = [('wazuh-authd', None, True), ('wazuh-db', None, True)]
+receiver_sockets, monitored_sockets, log_monitors = None, None, None  # Set in the fixtures
+
+# Functions
+def validate_authd_logs(logs):
+    for log in logs:
+        log_monitor.start(timeout=AUTHD_KEY_REQUEST_TIMEOUT,
+                          callback=make_callback(log, prefix=AUTHD_DETECTOR_PREFIX,
+                                                 escape=True),
+                          error_message='Expected error log does not occured.')
+
+def get_temp_force_config(param):
+    """
+    Creates a temporal config file.
+    """
+    temp = os.path.join(data_path, 'temp.yaml')
+    force_conf = {'force': {'elements': []}}
+    legacy_force_insert_conf = None
+    for elem in param:
+        if elem == 'force_insert':
+            legacy_force_insert_conf = {'force_insert': {'value': param[elem]}}
+        elif elem == 'disconnected_time':
+            disconnected_time_conf = {'disconnected_time':{'value': 0, 'attributes':[{'enabled':'no'}]}}
+            disconnected_time_conf['disconnected_time']['value'] = param[elem]['value']
+            disconnected_time_conf['disconnected_time']['attributes'][0]['enabled'] = param[elem]['attributes'][0]['enabled']
+            force_conf['force']['elements'].append(disconnected_time_conf)
+        else:
+            force_conf['force']['elements'].append({elem: {'value': param[elem]}})
+
+    with open(configurations_path, 'r') as conf_file:
+        temp_conf_file = yaml.safe_load(conf_file)
+        temp_conf_file[0]['sections'][0]['elements'].append(force_conf)
+        if legacy_force_insert_conf:
+            temp_conf_file[0]['sections'][0]['elements'].append(legacy_force_insert_conf)
+    with open(temp, 'w') as temp_file:
+        yaml.safe_dump(temp_conf_file, temp_file)
+    return temp
```

Comment:
IMHO this function should be split into the following function/fixtures:
- create_force_config_block: Create the config block following configure_environment fixture logic:
```
[{
   "section":"auth",
   "elements":[
      {
         "disabled":{
            "value":"no"
         }
      },
      {
         "port":{
            "value":1515
         }
      },
      {
         "use_source_ip":{
            "value":"no"
         }
      },
      {
         "force_insert":{
            "value":"yes"
         }
      },
      {
         "force_time":{
            "value":0
         }
      },
      {
         "purge":{
            "value":"yes"
         }
      },
      {
         "use_password":{
            "value":"no"
         }
      },
      {
         "limit_maxagents":{
            "value":"yes"
         }
      },
      {
         "ciphers":{
            "value":"HIGH:!ADH:!EXP:!MD5:!RC4:!3DES:!CAMELLIA:@STRENGTH"
         }
      },
      {
         "ssl_verify_host":{
            "value":"no"
         }
      },
      {
         "ssl_manager_cert":{
            "value":"/var/ossec/etc/sslmanager.cert"
         }
      },
      {
         "ssl_manager_key":{
            "value":"/var/ossec/etc/sslmanager.key"
         }
      },
      {
         "ssl_auto_negotiate":{
            "value":"no"
         }
      }
   ]
}]
```
- override_authd_force_conf (existing fixture): Use `configuration.set_section_wazuh_conf` function to set  configuration. Also, this fixture should backup the existing configuration using  `conf.get_wazuh_conf()` and set it after yield as follows:
```
    yield
    configuration.write_wazuh_conf(backup_config)
```